string command
string plan_name
string high_level_plan
